pkg/apis/certificates/types.go

```diff
@@ -42,6 +42,22 @@ type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
 	Request []byte
 
+	// Requested signer for the request.  It is a qualified name in the form: `scope-hostname.io/name`.
```

Comment:
Please linewrap comments at 80 characters like the rest of this file. There's a lot of double spaces in this comment.